project.clj

```diff
@@ -0,0 +1,34 @@
+(def tk-version "1.4.1")
+(def ks-version "1.3.1")
+
+(defproject puppetlabs/trapperkeeper-filesystem-watcher "0.1.0-SNAPSHOT"
+  :description "Trapperkeeper filesystem watcher service"
+  :url "https://github.com/puppetlabs/trapperkeeper-filesystem-watcher"
+  :license {:name "Apache License, Version 2.0"
+            :url "http://www.apache.org/licenses/LICENSE-2.0"}
+
+  :pedantic? :abort
+
+  :exclusions [org.clojure/clojure]
+
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/tools.logging "0.3.1"]
+                 [prismatic/schema "1.1.1"]
+                 [me.raynes/fs "1.4.6"]
+                 [puppetlabs/trapperkeeper ~tk-version]
+                 [puppetlabs/kitchensink ~ks-version]
+                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
```

Comment:
hey, ya know what's fun?

1. we've only ever done a single release of tk-sched.  yey?
2. there's no tag for it on github.  booo.    https://github.com/puppetlabs/trapperkeeper-scheduler/releases